supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesBackendSingletonActor.scala

```diff
@@ -3,19 +3,21 @@ package cromwell.backend.impl.jes
 import akka.actor.{Actor, ActorLogging, Props}
 import cromwell.core.Dispatcher.BackendDispatcher
 import cromwell.backend.impl.jes.statuspolling.JesApiQueryManager
-import cromwell.backend.impl.jes.statuspolling.JesApiQueryManager.DoPoll
+import cromwell.backend.impl.jes.statuspolling.JesApiQueryManager.JesApiQueryManagerRequest
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.numeric.Positive
 
-final case class JesBackendSingletonActor(qps: Int) extends Actor with ActorLogging {
+final case class JesBackendSingletonActor(qps: Int Refined Positive) extends Actor with ActorLogging {
```

Comment:
Hmm, I think I prefer leaving it raw but only because
 - I tend to dislike typedefs too
 - Putting on my evangelizing hat, people are more likely to see that it's an awesome general purpose library in this raw form.

I'll leave this up to you... (with silence meaning "probably won't change")